tests/test-team/template-mgmt-component-tests/template-mgmt-choose-page.component.ts

```diff
@@ -0,0 +1,135 @@
+import { test, expect } from '@playwright/test';
+import { TemplateMgmtChoosePage } from '../pages/template-mgmt-choose-page';
+
+test.describe('Choose Template Type Page', () => {
+  test('should land on "Choose Template Type" page when navigating to "/choose-a-template-type" url with empty session', async ({
+    page,
+    baseURL,
+  }) => {
+    const chooseTemplatePage = new TemplateMgmtChoosePage(page);
+
+    await chooseTemplatePage.navigateToChooseTemplatePage(
+      TemplateMgmtChoosePage.emptySessionData.id
+    );
+
+    await expect(page).toHaveURL(
+      `${baseURL}/templates/choose-a-template-type/3d98b0c4-6666-0000-0000-95eb27590000`
```

Comment:
This is great!

I wonder if we could use the Id directly from the `sessionWithNoData` object so that it's a bit easier to tie this Id back to the scenario? 

Otherwise it just looks like a magic number (uuid) with no meaning.